src/snyk/common/vscode/types.ts

```diff
@@ -56,3 +57,5 @@ export type ConfigurationParams = lsc.ConfigurationParams;
 export type CancellationToken = lsc.CancellationToken;
 export type ConfigurationRequestHandlerSignature = lsc.ConfigurationRequest.HandlerSignature;
 export type ResponseError<D = void> = lsc.ResponseError<D>;
+export type DidChangeConfigurationParams = lsc.DidChangeConfigurationParams;
+export const DID_CHANGE_CONFIGURATION_METHOD = 'workspace/didChangeConfiguration'; //lsc.DidChangeConfigurationNotification.method;
```

Comment:
You can use `lsc.DidChangeConfigurationNotification.type` as a type definition instead.